DRAFT_COROLLARY.md

```diff
@@ -0,0 +1,338 @@
+# Add missing constructors and assignment for `indirect` and `polymorphic`
+ <meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
+
+ISO/IEC JTC1 SC22 WG21 Programming Language C++
+
+D3152R0
+
+Working Group: Library Evolution, Library
+
+Date: 2024-03-18
+
+_Jonathan Coe \<<jonathanbcoe@gmail.com>\>_
+
+_Antony Peacock \<<ant.peacock@gmail.com>\>_
+
+_Sean Parent \<<sparent@adobe.com>\>_
+
+# Contents
+ - [Introduction](#introduction)
+ - [Additional constructors](#additional-constructors)
+ - [Converting assignment](#converting-assignment)
+ - [Technical specifications](#technical-specifications)
+ - [Reference implementation](#reference-implementation)
+ - [Acknowledgements](#acknowledgements)
+ - [References](#references)
+
+[//]: <> (<=============================================================================>)
+
+## Introduction
+
+New vocabulary types `indirect` and `polymorphic` for composite class design are
+proposed in P3019 [1]. Based on recommendations from LEWG, we follow up this
+work by adding support for converting construction, initializer-list
+construction and converting assignment to `indirect` and `polymorphic` where
+appropriate.
+
+## Additional constructors
+
+### Converting constructors
+
+We add converting constructors to both indirect and polymorphic so that they can
+be constructed from single values without the need to use `in_place` or
+`in_place_type` in line with optional and variant. As `indirect` and
+`polymorphic` are allocator-aware types, we provide allocator-extended versions
+of these constructors in line with those from `basic_optional` [2] and
+existing constructors from `indirect` and `polymorphic`.
+
+As `indirect` and `polymorphic` will use dynamic memory, the converting
+constructors are marked as explicit, the same as other constructors in
+`indirect` and `polymorphic`.
+
+### Initializer-list constructors
+
+We add initializer-list constructors to both `indirect` and `polymorphic` in
+line with those in optional and variant. As `indirect` and `polymorphic` are
+allocator-aware types, we provide allocator-extended versions of these
+constructors in line with those from `basic_optional` [2] and existing
+constructors from `indirect` and `polymorphic`.
+
+As `indirect` and `polymorphic` will use dynamic memory, the initializer-list
+constructors are marked as explicit, the same as other constructors in
+`indirect` and `polymorphic`.
+
+## Converting assignment
+
+We add a converting assignment for `indirect` in line with the converting
+assignment operators from optional and variant.
+
```

Comment:
Should this have a subsection heading e.g. Example ?

Reply:
Seems too short to justify it.